### DISCUSSION TITLE
Option to remove AI from Google search

### DISCUSSION THREAD

**Discussion by ghost** [benbusby/whoogle-search#1157] [Ideas]
AI is here to stay, regardless of our individual opinions of its quality, usefulnes, pro's and con's. It would be nice to have a choice in this context, so I'm wondering if this is possible to implement.

According to [this blog](https://blog.zgp.org/custom-google-search/) all it takes to remove AI from Google Search is including `udm=14` as a URL parameter.

Regards

**Comment by vinayyyr** [comment]
The idea of using udm=14 as a URL parameter to exclude AI-driven results in Google Search is feasible if Google implements it. Users could automate this via browser extensions or custom search engines. However, it depends on Google’s willingness to support such a feature, as filtering AI content is complex. Consider providing feedback to Google or exploring alternative search engines like DuckDuckGo for more control
